Sources/Utils/RequestModelBeautifier.swift

```diff
@@ -71,6 +70,29 @@ class RequestModelBeautifier: NSObject {
         txt.append("------------------------------------------------------------------------\n\n\n\n")
         return txt
     }
+
+    static func curlExport(request: RequestModel) -> String {
```

Comment:
this will only export request, right? maybe we can have it to export both request and curl and response in a separate file at the same time?

Reply:
That's correct, just the request. The same way the other request models work, essentially.

Reply:
@ilyapuchka ping ;)